diff --git a/debian/control b/debian/control
index 1eabec3..65b3389 100644
--- a/debian/control
+++ b/debian/control
@@ -51,7 +51,7 @@ Description: embeded lisp command interpreter library
 Package: librep-dev
 Section: libdevel
 Architecture: any
-Depends: rep, librep16 (>> ${source:Version}), ${shlibs:Depends}
+Depends: rep, librep16 (>= ${source:Version}), ${shlibs:Depends}
 Recommends: rep-doc
 Description: development libraries and headers for librep
  rep is a dialect of Lisp, designed to be used both as an extension
@@ -70,7 +70,7 @@ Package: librep-dbg
 Section: debug
 Priority: extra
 Architecture: any
-Depends: librep16 (>> ${source:Version}), ${shlibs:Depends}
+Depends: librep16 (>= ${source:Version}), ${shlibs:Depends}
 Recommends: rep (>> ${source:Version})
 Suggests: librep-dev, libncurses5-dbg, libreadline6-dbg | libreadline5-dbg, rep-doc
 Description: debug symbols for librep
@@ -83,7 +83,7 @@ Description: debug symbols for librep
 Package: rep-doc
 Section: doc
 Architecture: all
-Depends: info | info-browser, librep16 (>> ${source:Version}),
+Depends: info | info-browser, librep16 (>= ${source:Version}),
          dpkg (>> 1.15.4) | install-info
 Description: documentation for the lisp command interpreter
  rep is a dialect of Lisp, designed to be used both as an extension
